src/p7_config.h.in

```diff
@@ -131,5 +131,13 @@
  */
 #undef HAVE_FLUSH_ZERO_MODE
 
+/* On windows unsigned long int is 4 bytes, use unsigned long long for pointer arithmetic instead
```

Comment:
Why not use `uintptr_t` if available? The type is optional but the `configure` script should be able to detect it.

Reply:
I've updated `configure.ac` and `p7_config.h.in` to use the `uintptr_t` identified by autotools.